source/Playnite/ViewModels/SearchViewModel.cs

```diff
@@ -117,7 +117,7 @@ private bool GameFilter(Game game, string searchTerm, GameSearchFilterSettings s
                 return false;
             }
 
-            if (!SearchViewModel.MatchTextFilter(searchTerm, game.Name))
+            if (!SearchViewModel.MatchTextFilter(searchTerm, game.Name, mainModel.AppSettings.SearchBoxSearchWithAcronyms))

```

Comment:
This should be instead passed into the method as an argument, for easier testing in future.

Reply:
I don't understand since from my understanding that's already how it's being done, can you specify how it should be done if possible?

Reply:
Add new bool argument to GameFilter method and use that argument instead of direct reference to app settings.